src/popover/Popover.js

```diff
@@ -410,6 +411,36 @@ export default class Popover extends React.Component {
         anchorOffset.left = isHorizontal ? (child.width - popover.width) : 0;
         anchorOffset.top = isVertical ? (child.height - popover.height) : 0;
         break;
+      case 'auto':
+        _realAnchor = 'start';
```

Comment:
it would be great if the default remained `center` to avoid changing the UI of AlinityPRO.
do you think it would be more complicated?